ansible/roles/common/templates/conf/output/01-es.conf.j2

```diff
@@ -1,3 +1,17 @@
+{% if enable_caso | bool and inventory_hostname in groups['caso'] %}
+<match apel.events>
+    @type copy
+    <store>
+       @type elasticsearch
+       host { elasticsearch_address }}
+       port {{ elasticsearch_port }}
```

Comment:
```suggestion
       @type opensearch
       host { opensearch_address }}
       port {{ opensearch_port }}
```

Reply:
Fixed. I also replaced the other elasticsearch references in this file with opensearch, assuming they would follow the same format.